js/ad_manager_vast.js

```diff
@@ -1353,11 +1350,11 @@ OO.Ads.manager(function(_, $) {
           }
         }
 
-        var hasClickUrl = adWrapper.ad.data.linear.clickThrough.length > 0;
+        var clickUrl = _getLinearClickThroughUrl(adWrapper) || _getNonLinearClickThroughUrl(adWrapper);
         this.amc.notifyLinearAdStarted(adWrapper.id, {
           name: adWrapper.ad.data.title,
           duration: adWrapper.ad.durationInMilliseconds/1000,
-          hasClickUrl: hasClickUrl,
+          clickUrl: clickUrl,
```

Comment:
I've seen both boolean `hasClickUrl` and string `clickUrl` as properties here. I believe string `clickUrl` is the correct one?